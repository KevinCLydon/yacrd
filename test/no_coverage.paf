1	1000	100	150	+	2	100	0	100	100	100	255
