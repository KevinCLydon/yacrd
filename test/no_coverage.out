Not_covered:1,1000;850,150,1000;
